api/join/[token].ts

```diff
@@ -0,0 +1,90 @@
+import assert from 'assert';
```

Comment:
So we're using the same URL logic for both types of tokens, and just checking tokens against both tables in all places?

Reply:
yep!